resources/default_contacts.json

```diff
@@ -10,7 +10,35 @@
         },
         "photo-path": "icon_wallet_avatar",
         "add-chat?": true,
-        "dapp?": true
+        "dapp?": true,
+        "has-global-command?": true,
+        "dapp-url":
+        {
+          "en": "https://status.im/dapps/wallet/"
+        },
+        "bot-url": "local://wallet-bot"
```

Comment:
What feels weird in this file is that a local `bot-url` has `local://` schema while a local `photo-path` has `contacts://` schema. It's fine for now, but it's better to change `contacts://` to something more similar in the future.